### ISSUE TITLE
[TEST-FAILURE] CompletableFutureTest.postregisterTwoCallbacks_withFailure

### ISSUE THREAD

**Issue by bilalyasar** [hazelcast/hazelcast#5299]
```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 1
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:645)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:638)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:630)
	at com.hazelcast.executor.CompletableFutureTest.postregisterTwoCallbacks_withFailure(CompletableFutureTest.java:136)
```
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast/551/testReport/junit/com.hazelcast.executor/CompletableFutureTest/postregisterTwoCallbacks_withFailure/

**Comment by hasancelik** [comment]
```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds , count left: 1
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:718)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:711)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:703)
	at com.hazelcast.executor.CompletableFutureTest.postregisterTwoCallbacks_withFailure(CompletableFutureTest.java:126)

```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-OpenJDK8-Quality-Outreach/com.hazelcast$hazelcast/143/testReport/junit/com.hazelcast.executor/CompletableFutureTest/postregisterTwoCallbacks_withFailure/

**Comment by tombujok** [comment]
This one should be fixed already - via #6062 and #6106.

**Comment by gurbuzali** [comment]
let's close it then, if it fails again we can re-open
